iOS/Flows/Conversation/Cells/ConversationMessagesCell.swift

```diff
@@ -130,12 +157,41 @@ class ConversationMessagesCell: UICollectionViewCell {
 
         self.dataSource.shouldPrepareToSend = false
 
+        self.subscriptions.removeAll()
+
         // Remove all the items so the next message has a blank slate to work with.
         var snapshot = self.dataSource.snapshot()
         snapshot.deleteAllItems()
         self.dataSource.apply(snapshot, animatingDifferences: false)
     }
 
+    // MARK: - Update Subscriptions
+
+    func subscribeToUpdates() {
+        self.conversationController?
+            .messagesChangesPublisher
+            .mainSink { [unowned self] changes in
+                guard let conversationController = self.conversationController,
+                      let cid = conversationController.cid else { return }
+
+                var itemsToReconfigure: [MessageSequenceItem] = []
+                for change in changes {
+                    switch change {
+                    case .update(let message, _):
+                        itemsToReconfigure.append(.message(cid: cid, messageID: message.id))
+                    default:
+                        break
+                    }
+                }
```

Comment:
We need to mark which items need reconfiguration so that adding reactions still works.